content/en/docs/ops/configuration/security/security-policy-examples/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Security policy examples
+description: Shows common examples of using Istio security policy.
+weight: 60
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+## Background
+
+This page shows common patterns of using Istio security policy. You may find them useful in your deployment or use this
+as a quick reference to example policies.
+
+The policy demonstrated here are just examples and please make necessary changes to adapt to your actual environment
+before applying.
+
+Also read the [authentication tasks](/docs/tasks/security/authentication/authn-policy) and the
+[authorization tasks](/docs/tasks/security/authorization) for hands-on tutorial of using the security policy with much
+more details.
+
+### Require mandatory JWT validation
+
+The `RequestAuthentication` policy defines a list of JWT issuers that are allowed, by default it also allows
+requests without JWT token.
+
+After defining the `RequestAuthentication` policy, use the following extra policy to enforce mandatory JWT validation
```

Comment:
Do you want to add a RequestAuthentication policy example here?

Reply:
I prefer not, this page mainly lists example that is not found in the normal tasks page, the request authn example can be found in the task page already.

Reply:
Then add a link to the existing RequestAuthentication policy example.

Reply:
added.